locales-en-US.xml

```diff
@@ -66,16 +93,113 @@
     <term name="scale">scale</term>
     <term name="version">version</term>
 
-    <!-- ANNO DOMINI; BEFORE CHRIST -->
+    <!-- ITEM TYPES -->
+    <term name="article">preprint</term> <!-- see above question -->
+    <term name="article" form="short"></term>
+    <term name="article-journal">journal article</term>
+    <term name="article-journal" form="short"></term>
+    <term name="article-magazine">magazine article</term>
+    <term name="article-magazine" form="short"></term>
+    <term name="article-newspaper">newspaper article</term>
+    <term name="article-newspaper" form="short"></term>
+    <term name="bill">bill</term>
+    <term name="bill" form="short"></term>
+    <term name="book">book</term>
+    <term name="book" form="short"></term>
+    <term name="broadcast">broadcast</term>
+    <term name="broadcast" form="short"></term>
+    <term name="chapter">book chapter</term>
+    <term name="chapter" form="short"></term>
+    <term name="classic">classic</term>
+    <term name="classic" form="short"></term>
+    <term name="collection">collection</term>
+    <term name="collection" form="short"></term>
+    <term name="dataset">dataset</term>
+    <term name="dataset" form="short"></term>
+    <term name="document">document</term>
+    <term name="document" form="short"></term>
+    <term name="entry">entry</term>
+    <term name="entry" form="short"></term>
+    <term name="entry-dictionary">dictionary entry</term>
+    <term name="entry-dictionary" form="short"></term>
+    <term name="entry-encyclopedia">encyclopedia entry</term>
+    <term name="entry-encyclopedia" form="short"></term>
+    <term name="event">event</term>
+    <term name="event" form="short"></term>
+    <term name="figure">figure</term> <!-- there is an existing figure term right? -->
```

Comment:
@bwiernik Yes, under locator we already have a `figure` term. In other similar cases we've renamed the locator term to `X-locator`, as in `article` vs. `article-locator` or `title` vs. `title-locator`.

Reply:
@bwiernik What do you think about this? Would renaming the existing figure term to `figure-locator` make sense?

Reply:
@bwiernik What do you think about this? Should we rename it to `figure-locator`?

Reply:
We need terms for both `figure` and `figure-locator` (and `article` and `article-locator`)

Reply:
Ok, I've renamed the locator term to `figure-locator`. We have both terms for the locator and for the item type now.